holoviews/plotting/plot.py

```diff
@@ -920,6 +920,9 @@ def __init__(self, layout, keys=None, dimensions=None, **params):
                                                    **params)
         if top_level:
             self.comm = self.init_comm(layout)
+        self.streams = [s for streams in layout.traverse(lambda x: x.streams,
+                                                         [DynamicMap])
```

Comment:
The wrapping is a bit weird. I would put ``in layout.traverse(lambda x: x.streams, [DynamicMap])`` onto the second line and ``for s in streams`` onto the third line.